### ISSUE TITLE
List of Bugs from Playtesting Session 6-7-2021

### ISSUE THREAD

**Issue by MercuriusXeno** [MercuriusXeno/Goo#120]
In no particular order, split these up later.
1) Weird goo splat teleports port in the same frame or at least observably twice in short succession - the goo splats don't go on cooldown so the result is you get teleported immediately back to where you came from, jarring and not correct.
2) Weird goo blobs bounce right off their victims and fall harmlessly to the ground, they're supposed to teleport stuff randomly like an ender pearl.
3) For some reason the radial menu numbers vanish once the radial finishes opening completely.
4) Pump item frame renderer weirdly large, try to make it match what the solidifier now does, it looks like garbage.
[11:36 PM] MercuriusXeno: 5) JEI integration wasn't firing on some things for no apparent reason (bug was observed on a diamond)
6) JEI for Solidifier shows recipes marked as can't solidify
[11:51 PM] MercuriusXeno: 7) Weird visual (missing texture) happened while placing floral goo, not sure what texture it was
8) floral goo had weird desync issues when applying its bonemeal effect
9) floral goo got stuck in grass after #8 desync happened, weirdly floating in midair.
[11:54 PM] MercuriusXeno: 10) slime landing fall damage cushion apparently stopped working
11) Crash when hovering over a lava bucket and holding shift to view it's goo values! Something broken in the targeting handler.
